actuator-firmware/src/servo.c

```diff
@@ -100,6 +95,9 @@ THD_FUNCTION(servo_thd, arg)
 
 void servo_start(void)
 {
-    static THD_WORKING_AREA(servo_thd_wa, 256);
+    static THD_WORKING_AREA(servo_thd_wa, 512);
```

Comment:
We should monitor actual stack usage and not just randomly double.
This is ok for development but we might run into memory shortage on this constrained MCU.

Reply:
+1, not even sure that stack size is an issue here.